compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/util/Flags.java

```diff
@@ -225,6 +225,10 @@ public static int asMask(Set<Flag> flagSet) {
         return flagSet;
     }
 
+    public static int unset(int mask, int flag) {
```

Comment:
not used ?

Reply:
This is used in `desugarTemplateLiteralObjectTypedef()` method in Desugar